File: Hello.java
import java.util.Scanner;
public class Hello{
 public static void main(String args[]){
  System.out.print("Enter your name: ");
  Scanner keyboard=new Scanner(System.in);
  String Name = keyboard.nextLine(); 
  System.out.print("Hello world");
  System.out.println(", I am "+Name);
  
 }
}

File: DjangoMinimalist/hello.py
"""
This is a single-file hello world file using django!
To see this in action, place this file into your project directory and run
the following command:

$python hello.py runserver

And visit localhost:8000 in your browser to see your single-file hello world!
"""

import sys
from django.conf import settings


settings.configure(
    DEBUG=True,
    SECRET_KEY='thisisthesecretkey',
    ROOT_URLCONF=__name__,
    MIDDLEWARE_CLASSES=(
        'django.middleware.common.CommonMiddleware',
        'django.middleware.csrf.CsrfViewMiddleware',
        'django.middleware.clickjacking.XFrameOptionsMiddleware',
    ),
)


from django.conf.urls import url
from django.http import HttpResponse


def index(request):
    return HttpResponse('Hello World')


urlpatterns = (
    url(r'^$', index),
)


if __name__ == "__main__":
    from django.core.management import execute_from_command_line

    execute_from_command_line(sys.argv)
  
